automation/src/test/scala/org/broadinstitute/dsde/test/api/RawlsApiSpec.scala

```diff
@@ -247,7 +247,7 @@ class RawlsApiSpec
       }(owner.makeAuthToken(billingScopes))
     }
 
-    "should be able to create workspace and run sub-workflow tasks in non-US regions" taggedAs(MethodsTest) in {
+    "should be able to create workspace and run sub-workflow tasks in non-US regions" taggedAs(MethodsTest) ignore in {
```

Comment:
change `ignore in` to just `ignore`, I think:
```suggestion
    "should be able to create workspace and run sub-workflow tasks in non-US regions" taggedAs(MethodsTest) ignore {
```